### ISSUE TITLE
Previously pinned tabs are no longer pinned

### ISSUE THREAD

**Issue by DamirTomic** [Drive4ik/simple-tab-groups#1066]
**Describe the bug**
Starting from last week, previously pinned tabs are no longer pinned. Even if I pin a tab, when I close and reopen firefox, they will all be unpinned and simple-tab-groups will do some strange tab scrolling before fully loading everything

**To Reproduce**
1. pin a tab
2. close firefox
3. re-open firefox
4. pinned tab is gone

**Expected behavior**
1. pin a tab
2. close firefox
3. re-open firefox
4. pinned tabs should remain
 
**Screenshots**

**Desktop (please complete the following information):**
 Windows 10
Firefox 113.0.2

**Comment by DamirTomic** [comment]
The issue seems to have resolved itself after updating to firefox 114.0

**Comment by mmmaiden** [comment]
No, the issue did not resolve itself. I can confirm pinned tabs are NEVER restored, no matter which firefox version.
**Desktop**: openSUSE Tumbleweed, Firefox 116.0.2 (64-Bit)
I mentioned this issue some month ago (#1001)  but nothing changed since then; I still have to load a backup EVERY DAY when starting firefox, because the pinned tabs NEVER show up on start. All other tabs and groups are showing after starting FF for the first time of the day, the pinned tabs don't ever. 
I tried the following:  opened a existing backup in an editor, deleted all groups so that only general settings and my pinned tabs are left, saved and then load this backup, but no pinned tabs are loaded.
I REALLY like this extension, but having to manually load a backup every day is just awkward and annoying.
